src/components/transactions/TxDetails/TxData/DecodedData/Multisend/index.tsx

```diff
@@ -90,7 +90,7 @@ export const Multisend = ({
       {!noHeader && <MultisendActionsHeader setOpen={setOpenMap} amount={multiSendTransactions.length} />}
 
       <div className={noHeader && multiSendTransactions.length >= MIN_SCROLL_TXS ? css.scrollWrapper : undefined}>
-        <Box display="flex" flexDirection="column" gap={1}>
+        <Box display="flex" flexDirection="column">
```

Comment:
This gap was there for a reason. It’s needed for the DecodedTx situation. Could you move it there?

Reply:
I've added it when `noHeader` is present:

![image](https://user-images.githubusercontent.com/20442784/234859165-9c27d463-5431-4ec0-9033-68f26a9608f6.png)

Reply:
Thanks!